services/appveyor/appveyor-build-redirect.service.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const { redirector } = require('..')
+
+module.exports = [
+  redirector({
+    category: 'build',
+    route: {
+      base: 'appveyor/ci',
```

Comment:
In other services/platforms where we provide multiple CI-related like badges build, test, coverage (Azure DevOps, Jenkins, TeamCity, etc.) we've been using `build`, `tests`, `coverage`, so I figured I'd change appveyor to use `build` instead of `ci` to be consistent.